addons/medical/functions/fnc_fullHeal.sqf

```diff
@@ -0,0 +1,26 @@
+#include "..\script_component.hpp"
+/*
+ * Author: DartRuffian
+ * Fully heals the patient.
+ *
+ * Arguments:
+ * 0: Medic <OBJECT>
+ * 1: Patient <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, cursorObject] call ace_medical_fnc_fullHeal
+ *
+ * Public: Yes
+ */
+
+params [["_medic", objNull, [objNull]], ["_patient", objNull, [objNull]]];
+
+if (isNull _medic or !alive _patient) exitWith {
+    ERROR_2("fullHeal [medic %1][patient %2] Bad parameters",_medic,_patient);
+};
+
+[_medic, _patient] call EFUNC(medical_treatment,fullHeal);
```

Comment:
Thought about adding a boolean flag to fullHeal to enable/disable the full heal message that shows in the medical GUI. Should that be a separate PR after this or just do it in one?